### ISSUE TITLE
Delta V calculation in VAB sometimes gets confused if root part is not in final stage

### ISSUE THREAD

**Issue by OJezu** [MuMech/MechJeb2#983]
It looks like delta V calculation includes mass of stages below the calculated stage. Adding and removing parts to earlier stages causes changes to calculation on latter stages. Even if the root part is in those.

I've seen it happen, when root part is not in the final (0) stage. Re-rooting the rocket a few times seems to fix the issue.

E.g. Root part is HECS2 in stage 1, visible on to of the screenshots. Also in stage 1 there is some fuel and an engine.  Stage 0 is a rover, no fuel there. Adding parts below stage 1 engine causes delta V calculation for stage 1 to be thrown off (3000 vs 700).

This particular rocket was build with a bit of re-rooting assemblies, so that might be the key to recreating the issue. Craft file attached, along with screenshots with parts below either attached or detached.

![mechjeb 2 7 delta v 2](https://user-images.githubusercontent.com/3941091/35934066-c00fbe30-0c3c-11e8-8b21-96122c7b473f.PNG)
![mechjeb 2 7 delta v](https://user-images.githubusercontent.com/3941091/35934055-b950f9d8-0c3c-11e8-993a-d858922595fd.PNG)

[UR 1 + d30 + d39.zip](https://github.com/MuMech/MechJeb2/files/1704268/UR.1.d30.d39.zip)

**Comment by lamont-granquist** [comment]
Yeah, this is a requirement of using the delta V analyzer and is tightly coupled with the entire design.
